src/algebra/module/linear_map.lean

```diff
@@ -200,33 +235,45 @@ variable {R}
 f.to_add_monoid_hom.map_sum _ _
 
 theorem to_add_monoid_hom_injective :
-  function.injective (to_add_monoid_hom : (M →ₗ[R] M₂) → (M →+ M₂)) :=
+  function.injective (to_add_monoid_hom : (M →ₛₗ[σ] M₃) → (M →+ M₃)) :=
 λ f g h, ext $ add_monoid_hom.congr_fun h
 
 /-- If two `R`-linear maps from `R` are equal on `1`, then they are equal. -/
```

Comment:
Should we be purging "`R`-linear" from docstrings in this file in favor of "`σ`-semilinear"?